### ISSUE TITLE
Rework TextExporter [438]

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/OpenMS#438]
Submitted by cbielow on 2012-07-26 10:52:19

the current output format and the plethora of options are just overwhelming and the resulting CSV's are hard to parse, since they can contain interspersed Feature,Peptide and Protein data.

We could do it the MaxQuant way:
- write a table for each entity (proteins, peptides, features)
- tables are linked via an ID column.

This enables easy parsing and assignment in R.
Might be harder to read manually, because the visual link is lost.

**Comment by jpfeuffer** [comment]
In the course of reworking the TextExporter we have to think about the handling of the protein group information which is currently not supported.

**Comment by timosachsenberg** [comment]
why not use mzTab?

**Comment by timosachsenberg** [comment]
Maybe we should sit down and take a look how we can encode this in mzTab. We already have protein and PSM columns (and something like linked peptide columns) in mzTab that can be linked using an additional opt_ column

**Comment by jpfeuffer** [comment]
Yes, sure. I forgot about mzTabExporter, while using an old workflow.
Let us continue it there.

**Comment by timosachsenberg** [comment]
Would be great. Hope this will iron out some undetected bugs in the exporter.

**Comment by jpfeuffer** [comment]
Let's close this and focus on mzTab instead.
